### PR TITLE
Commerce 1735 Add backend for unit price to be updated on placed orders in Admin UI

### PR DESCRIPTION
**Pull request by alecsloan**

### CODE REVIEW

**Comment by alecsloan** [comment]
ci:test

**Comment by marco-leo** [comment]
@alecsloan please rebase this

**Comment by alecsloan** [comment]
> @alecsloan please rebase this

@marco-leo Rebased

File: commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/portlet/action/EditCommerceOrderItemMVCActionCommand.java [hunk 1]
```diff
@@ -149,12 +153,20 @@ protected void updateCommerceOrderItem(ActionRequest actionRequest)
 				commerceOrderItemId, quantity, commerceContext, serviceContext);
 		}
 		else {
-			_commerceOrderItemService.updateCommerceOrderItem(
-				commerceOrderItemId, quantity, commerceOrderItem.getJson(),
-				commerceContext, serviceContext);
+			BigDecimal price = (BigDecimal)ParamUtil.getNumber(
+				actionRequest, "price");
+
+			CommerceMoney unitPrice = _commerceMoneyFactory.create(
```
Comment (marco-leo) [inline commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/portlet/action/EditCommerceOrderItemMVCActionCommand.java]: @alecsloan  I don't need we have to convert in a CommerceMoney first

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
needs to re-tested after #1494 is merged

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Comment by alecsloan** [comment]
ci:test

**Pull request merged**